docs/source/conf.py

```diff
@@ -12,7 +12,20 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../../python'))
+import subprocess
```

Comment:
Is there a reason this is using `os` and not `pathlib` here?

Reply:
Don't really care, I just personally find `pathlib` a bit easier to read. Totally up to you, just figured I'd mention it in case you weren't aware of it :slightly_smiling_face: